api/apitestlib.go

```diff
@@ -34,31 +34,26 @@ type TestHelper struct {
 }
 
 func setupTestHelper(enterprise bool) *TestHelper {
+	utils.TranslationsPreInit()
+	utils.LoadConfig("config.json")
+	utils.InitTranslations(utils.Cfg.LocalizationSettings)
+
 	th := &TestHelper{
 		App: app.New(),
 	}
 
-	if th.App.Srv == nil {
-		utils.TranslationsPreInit()
-		utils.LoadConfig("config.json")
-		utils.InitTranslations(utils.Cfg.LocalizationSettings)
-		*utils.Cfg.TeamSettings.MaxUsersPerTeam = 50
-		*utils.Cfg.RateLimitSettings.Enable = false
-		utils.Cfg.EmailSettings.SendEmailNotifications = true
-		utils.DisableDebugLogForTest()
-		th.App.NewServer()
-		th.App.InitStores()
-		th.App.Srv.Router = NewRouter()
-		th.App.Srv.WebSocketRouter = th.App.NewWebSocketRouter()
-		th.App.StartServer()
-		api4.Init(th.App, th.App.Srv.Router, false)
-		Init(th.App, th.App.Srv.Router)
-		wsapi.Init(th.App, th.App.Srv.WebSocketRouter)
-		utils.EnableDebugLogForTest()
-		th.App.Srv.Store.MarkSystemRanUnitTests()
+	*utils.Cfg.TeamSettings.MaxUsersPerTeam = 50
+	*utils.Cfg.RateLimitSettings.Enable = false
+	utils.Cfg.EmailSettings.SendEmailNotifications = true
+	utils.DisableDebugLogForTest()
+	th.App.StartServer()
+	api4.Init(th.App, th.App.Srv.Router, false)
+	Init(th.App, th.App.Srv.Router)
+	wsapi.Init(th.App, th.App.Srv.WebSocketRouter)
+	utils.EnableDebugLogForTest()
+	th.App.Srv.Store.MarkSystemRanUnitTests()
```

Comment:
This is still too much initialization code, and I plan to streamline more of it, but I'm really aiming to get the tests running with ramdisk and/or NoSQL stores early next week. So that'll come in a later PR.